src/common/libzmqutil/test/msg_zsock.c

```diff
@@ -36,8 +39,9 @@ void check_sendzsock (void)
         && zmq_connect( zsock[1], uri) == 0,
         "got inproc socket pair");
 
-    zsock_set_linger (zsock[0], 5);
-    zsock_set_linger (zsock[1], 5);
+    if (zsetsockopt_int (zsock[0], ZMQ_LINGER, 5) < 0
+        || zsetsockopt_int (zsock[1], ZMQ_LINGER, 5) < 0)
+        BAIL_OUT ("could not set ZMQ_LINGER socket option");
 
     if (!(any = flux_msg_create (FLUX_MSGTYPE_ANY)))
```

Comment:
unrelated to this PR, but running this test under valgrind showed that `any` is leaked here.